src/main/kotlin/frc/robot/RobotContainer.kt

```diff
@@ -32,24 +33,29 @@ object RobotContainer {
     }
 
     private fun configureDefaultCommands() {
-        swerveDrive.defaultCommand = DriveCommands.joystickDrive(
+        swerveDrive.defaultCommand = DriveCommands.joystickDriveAtAngle(
             swerveDrive,
-            { MathUtil.applyDeadband(driverController().leftY, 0.15) },
-            { MathUtil.applyDeadband(driverController().leftX, 0.15) },
-            { 0.7 * MathUtil.applyDeadband(-driverController().rightX, 0.15) }
-        )
+            { driverController.leftY },
+            { driverController.leftX },
+            { swerveDrive.desiredHeading }
+        ).alongWith(swerveDrive.updateDesiredHeading { -driverController.rightX })
     }
 
     private fun configureButtonBindings() {
-        driverController().y().onTrue(
+        driverController.y().onTrue(
             Commands.runOnce({
                 swerveDrive.pose = Pose2d(swerveDrive.pose.translation, Rotation2d())
             }, swerveDrive)
                 .ignoringDisable(true)
         )
+
+        driverController.povUp().whileTrue(swerveDrive.setDesiredHeading(Rotation2d.fromDegrees(0.0)))
+        driverController.povRight().whileTrue(swerveDrive.setDesiredHeading(Rotation2d.fromDegrees(-90.0)))
+        driverController.povDown().whileTrue(swerveDrive.setDesiredHeading(Rotation2d.fromDegrees(180.0)))
+        driverController.povLeft().whileTrue(swerveDrive.setDesiredHeading(Rotation2d.fromDegrees(90.0)))
```

Comment:
Use the new constants from here
https://github.wpilib.org/allwpilib/docs/development/java/edu/wpi/first/math/geometry/Rotation2d.html#kZero
This will help reduce memory usage. Should be changed anywhere else in the code as well.

Reply:
4895ee0a8b23e9e2207d221e1679ad3ccc6aaccc